app/lib/pay-request/client.js

```diff
@@ -24,7 +24,8 @@ const buildPayBaseClient = function buildPayBaseClient (service) {
     maxContentLength: 5 * 1000 * 1000,
 
     // @TODO(sfount) configure headers based on each services requirements
-    // headers: { 'X-Auth-header': 'secretscontent' }
+    headers:
```

Comment:
Is this header not the same for other services? I'm not sure why this would be different or we should consider how this pattern could scale